src/shared/lib/MutationAnnotator.ts

```diff
@@ -63,12 +63,46 @@ export async function annotateMutations(mutations: Mutation[],
         }
     ): [];
 
-    const indexedVariantAnnotations = indexAnnotationsByGenomicLocation(variantAnnotations);
+    return indexAnnotationsByGenomicLocation(variantAnnotations);
```

Comment:
needed this result in the mutation mapper store generator, such that the mutations could be filtered by transcript. I think it's good to keep both the mutations and the annotations available such that we can add the dropdown box for transcript selection at some point